src/NHSD.BuyingCatalogue.Database/PostDeployment/ProdLikeData/MergeSolutions.sql

```diff
@@ -0,0 +1,321 @@
+
+MERGE INTO [dbo].[Solution] AS TARGET
+USING #Solution AS SOURCE
+ON TARGET.[Id] = SOURCE.[Id] 
+WHEN MATCHED THEN  
+    UPDATE SET TARGET.[Version] = SOURCE.[Version],
+                TARGET.[Summary] = SOURCE.[Summary],
+                TARGET.[FullDescription] = SOURCE.[FullDescription],
+                TARGET.[Features] = SOURCE.[Features],
+                TARGET.[ClientApplication] = SOURCE.[ClientApplication],
+                TARGET.[Hosting] = SOURCE.[Hosting],
+                TARGET.[ImplementationDetail] = SOURCE.[ImplementationDetail],
+                TARGET.[RoadMap] = SOURCE.[RoadMap],
+                TARGET.[IntegrationsUrl] = SOURCE.[IntegrationsUrl],
+                TARGET.[AboutUrl] = SOURCE.[AboutUrl],
+                TARGET.[ServiceLevelAgreement] = SOURCE.[ServiceLevelAgreement],
+                TARGET.[WorkOfPlan] = SOURCE.[WorkOfPlan],
+                TARGET.[LastUpdated] = SOURCE.[LastUpdated],
+                TARGET.[LastUpdatedBy] = SOURCE.[LastUpdatedBy]
+WHEN NOT MATCHED BY TARGET THEN  
+    INSERT  ([Id], [Version], [Summary], [FullDescription], [Features], [ClientApplication], [Hosting], [ImplementationDetail], [RoadMap], [IntegrationsUrl], [AboutUrl], [ServiceLevelAgreement], [WorkOfPlan], [LastUpdated], [LastUpdatedBy]) 
+    VALUES  (SOURCE.[Id], SOURCE.[Version], SOURCE.[Summary], SOURCE.[FullDescription], SOURCE.[Features], SOURCE.[ClientApplication], SOURCE.[Hosting], SOURCE.[ImplementationDetail], SOURCE.[RoadMap], SOURCE.[IntegrationsUrl], SOURCE.[AboutUrl], SOURCE.[ServiceLevelAgreement], SOURCE.[WorkOfPlan], SOURCE.[LastUpdated], SOURCE.[LastUpdatedBy]);
+END;
+GO
```

Comment:
I think it might be better to consolidate the various single-line solution inserts that are currently spread throughout the file as a single `INSERT` statement. What do you think?

Reply:
These were generated from SQL Server. I think I'd rather leave as is, so they can easily be replaced again.